drivers/flash/Kconfig.nrf

```diff
@@ -27,6 +27,37 @@ config SOC_FLASH_NRF_RADIO_SYNC
 	help
 	  Enable synchronization between flash memory driver and radio.
 
+if HAS_HW_NRF_NVMC_PE
```

Comment:
Not needed. `SOC_FLASH_NRF_PARTIAL_ERASE` depends on `HAS_HW_NRF_NVMC_PE` and `SOC_FLASH_NRF_PARTIAL_ERASE_MS` depends on `SOC_FLASH_NRF_PARTIAL_ERASE`, so you already got the required dependency rules in place.

Reply:
OK